recipes/neuralprophet/LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+Copyright <2023> <MIT>
```

Comment:
The license is in the tarball and you don't need this file.

Reply:
ok ill delete this as well

Reply:
thanks should be all good now